src/bust.py

```diff
@@ -325,6 +326,40 @@ def get_google_form_url(self, image_url: Optional[str] = None) -> Optional[str]:
         )
         return form_url
 
+    async def send_stats(self, interaction: Interaction):
+        songs_len = int(self.total_song_len)
+        num_songs = len(self.current_channel_content)
+        bust_len = songs_len + config.seconds_between_songs * num_songs
+
+        # Compute map submitter --> total length of all submissions
```

Comment:
```suggestion
        # Compute map of submitter --> total length of all submissions
```

nit: it took me a second to parse in my head what "Compute map submitter" meant.